examples/mobile/UIComponents/components/progress/progress-bar-appbar/progressbar.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+	<meta content="width=device-width, user-scalable=no" name="viewport" />
+	<link href="../../../lib/tau/mobile/theme/default/tau.css" rel="stylesheet" />
+	<link href="../../../css/style.css" rel="stylesheet" />
+	<script data-build-remove="false" src="../../../lib/tau/mobile/js/tau.js"></script>
+	<title>Progress Bar</title>
+</head>
+
+<body>
+	<div class="ui-page" id="progressbar-appbar-demo">
```

Comment:
I think it would be good to sync with file name.
id="progress-bar-app-demo" following the file name
or
change the folder name to the "progress-bar-appbar".

Reply:
Done.